app/assets/javascripts/components/Columns/ColumnItem.js

```diff
@@ -1,20 +1,46 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from 'react'
+import { useDrop } from 'react-dnd';
+import PropTypes from 'prop-types'
 
-const Column = ({ title, children, renderAction, onClose }) => (
-  <div className="Column">
-    <div className="Column__header">
-      <h3 className="Column__name">{title}</h3>
-      <div className="Column__actions">
-        {renderAction()}
-        <button type="button" className="Column__btn-close" onClick={onClose}>
-          <i className="mi md-light md-16">close</i>
-        </button>
+const type = {
+  story: 'STORY'
+}
+
+const Column = ({
+  title,
+  children,
+  renderAction,
+  onClose,
+  column
+}) => {
+  const [, drop] = useDrop({
+    accept: type.story,
+    canDrop() {
```

Comment:
We can create one method in `Column` model. We already have `const done` in model file.

`export const isDone = column => column !== done`

And here, we change to:

`canDrop: Column.isDone`